presentation/src/main/java/com/knocklock/presentation/navigation/KnockLockNavHost.kt

```diff
@@ -128,9 +139,55 @@ fun NavGraphBuilder.settingGraph(
             )
         }
         composable(route = NavigationRoute.SettingGraph.Credit.route) {
+            val context = LocalContext.current
             CreditRoute(
                 modifier = modifier,
-                onIconClick = { navController.popBackStack() }
+                onIconClick = { navController.popBackStack() },
+                onTextClicked = { textRes ->
+                    when (textRes) {
+                        R.string.github -> {
+                            val intent = Intent(Intent.ACTION_VIEW).apply {
+                                data = Uri.parse(GITHUB_LINK)
+                            }
+                            context.startActivity(intent)
+                        }
+                        R.string.inquiry -> {
+                            val intent = Intent(Intent.ACTION_SENDTO).apply {
+                                data = Uri.parse("mailto:")
+                                putExtra(Intent.EXTRA_EMAIL, arrayOf(KNOCK_LOCK_EMAIL_ADDRESS))
+                            }
+                            intent.resolveActivity(context.packageManager)?.let {
+                                context.startActivity(intent)
+                            }
```

Comment:
에뮬레이터에서 적절한 메일 앱이 없어서 클릭해도 아무 동작이 없네요.
query 명시를 안해서 그런가 했더니 아니더라구요.

공식 문서를 보니까 아래 코드처럼 구현하면 적절한 앱이 없다는 것도 UI로 표시되어서 좋은 것 같아요!
```suggestion
                             val intent = Intent(Intent.ACTION_SENDTO).apply {
                                data = Uri.parse("mailto:")
                                putExtra(Intent.EXTRA_EMAIL, arrayOf(KNOCK_LOCK_EMAIL_ADDRESS))
                            }
                            context.startActivity(Intent.createChooser(intent, null))
```

![image](https://user-images.githubusercontent.com/33657541/226948065-0078a576-cdf5-4494-84ef-0b705277b32e.png)